docs/extensibility/50-list-and-details-widgets.md

```diff
@@ -0,0 +1,620 @@
+The `FeaturedCard` can be closed in the top right corner.
+It is important to select a unique ID for each `FeaturedCard`, since it will be displayed/hidden based on its ID.
+
+**TIP:** It is **recommended** to use the `FeatureCard` widget via [injections](./70-widget-injection.md) (**slot: banner**). This will turn all `FeatureCards` at the top of a section into a carousel-component. Nevertheless, it is also possible to use it as a regular widget.
+
+These are the available `FeatureCard` widget parameters:
+
+| Parameter | Required | Type | Description |
+|-----------|----------|------|-------------|
+| **title** | | string | Renders the title. |
+| **description** | | string | Renders the description. |
+| **id** | | string | Defines the banner's **(unique!)** identifier. |
+| **children** | |  - an array of a maximum of 2 objects | Defines the banner's children that are displayed below the description. It is recommended to use button-like components here. |
+| **illustration** | | | Defines the illustration displayed on the right. Can be one of the following: `Modules` (default), `AI` |
+| **design** | | | Defines the background color-gradient of the banner. Can be one of the following: `information-1` (default), `information-2`. |
+
+See the following example:
+
+```yaml
+injections: |-
+  - title: Introducing Modules
+    description: Modules add functionalities to your cluster. Consume SAP BTP services, monitor your cluster, build serverless applications and more.
+    widget: FeaturedCard
+    id: ModulesBanner
+    illustration: Modules
+    design: information-1
+    children:
+      - widget: Wizard
+        name: Add Modules
+        wizard: module-wizard
+      - widget: ExternalLinkButton
+        link: https://help.sap.com/docs/btp/sap-business-technology-platform/kyma-s-modular-approach
+    order: 0
+    targets:
+      - slot: details-banner
+        location: ClusterOverview
+```
+
+<img src="./assets/display-widgets/FeaturedCard.png" alt="Example of a FeaturedCard widget" style="border: 1px solid #D2D5D9">
+
+### `Panel`
+
+The `Panel` widgets render an object as a separate panel with its own title (based on its `source` or `name`).
+
+These are the available `Panel` widget parameters:
+
+| Parameter | Required | Type | Description |
+|-----------|----------|------|-------------|
+| **header** | No | array | Allows you to, for example, display labels in the panel header. |
+| **disablePadding** | No | boolean | Disables the padding inside the panel body. |
+| **description** | | | Displays a custom description on the resource list page. It can contain links. If the **translations** section has a translation entry with the ID that is the same as the **description** string, the translation is used. |
+| **decodable** | | | Decodes the values of all the children using base64 - must be used together with the **source** parameter. |
```

Comment:
```suggestion
| **header** | No | array | Allows you to, for example, display labels in the panel header. |
| **disablePadding** | No | boolean | Disables the padding inside the panel body. |
| **description** | No | string | Displays a custom description on the resource list page. It can contain links. If the **translations** section has a translation entry with the ID that is the same as the **description** string, the translation is used. |
| **decodable** | No | boolean | Decodes the values of all the children using base64 - must be used together with the **source** parameter. |
```